src/Person/Person.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  let spouse;
+
+  if (sex === 'm') {
+    spouse = 'wife';
+  } else if (sex === 'f') {
+    spouse = 'husband';
+  }
```

Comment:
It's better to use ternary operator if you should define variable value depending on some check